tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+          type: integer
+          required: false
+        - in: query
+          name: take
+          description: Used for pagination. Specfies the number of entries to include in the response list. Typically used in combination with the skip parameter.
+          type: integer
+          required: false
+      responses:
+        200:
+          schema:
+            $ref: '#/responses/GetTagsResponse'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags-count:
+    get:
+      tags:
+        - tags
+      summary: Get the total number of tags
+      description: Get a count of the total number of tags.
+      operationId: GetTagCount
+      parameters: []
+      responses:
+        200:
+          description: Contains the number of tags
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags-with-values:
+    get:
+      tags:
+        - tags
+      summary: Get the current values of multiple tags
+      responses:
+        200:
+          schema:
+            $ref: '#/responses/GetTagsWithValuesResponse'
+          description: Success
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: Conflict
+        default:
+          $ref: '#/responses/Error'
+      description: 'Get a list of tags with their values in a single request, filtered based on the provided query parameters.'
+      operationId: GetTagsWithValues
+      parameters:
+        - in: query
+          name: path
+          description: Comma-separated list of tag paths. May include glob-style wildcards.
+          type: string
+          required: false
+        - in: query
+          name: keywords
+          description: Comma-separated list of case-sensitive keywords
+          type: string
+          required: false
+        - in: query
+          name: properties
+          description: Comma-separated list of name=value strings, e.g. key1=value,key2=value.
+          type: string
+          required: false
+        - in: query
+          name: skip
+          description: Used for pagination. Specifies the number of entries to skip in the response list. Typically used in combination with the take parameter.
+          type: integer
+          required: false
+        - in: query
+          name: take
+          description: Used for pagination. Specfies the number of entries to include in the response list. Typically used in combination with the skip parameter.
+          type: integer
+          required: false
+  /v2/tags-with-values/{path}:
+    get:
+      tags:
+        - tags
+      summary: Get a tag and its current value
+      description: Get a tag and its value
```

Comment:
What's the difference between returning a tag and returning the value of a tag? Metadata?

Also, this is another place where the description and summary seem switched.

Reply:
See my other response for the first question. And yeah, I swapped the summary and description.